Service/Json/JsonReader.php

```diff
@@ -66,18 +66,26 @@ class JsonReader implements \Countable, \Iterator
     private $convertDocuments;
 
     /**
+     * @var array
+     */
+    private $options;
+
+    /**
      * Constructor.
      *
      * @param Manager $manager
      * @param string  $filename
+     * @param array   $options
      * @param bool    $convertDocuments
+     *
      */
-    public function __construct($manager, $filename, $convertDocuments = true)
+    public function __construct($manager, $filename, $options, $convertDocuments = true)
```

Comment:
I took a bit deeper look and don't think we need this argument. Nothing in the whole bundle uses this option. And by current logic nothing will. Propose to remove `$convertDocuments`. And then remove `convertDocument()` function.